.github/workflows/qiita-ci.yml

```diff
@@ -118,12 +118,10 @@ jobs:
           COVER_PACKAGE: ${{ matrix.cover_package }}
         run: |
           conda activate qiita_client
-          export QIITA_SERVER_CERT=`pwd`/qiita-dev/qiita_core/support_files/server.crt
+          export QIITA_ROOTCA_CERT=`pwd`/qiita-dev/qiita_core/support_files/ci_rootca.crt
           export QIITA_CONFIG_FP=`pwd`/qiita-dev/qiita_core/support_files/config_test_local.cfg
-
-          nosetests --with-doctest --with-coverage --cover-package=qiita_client
-
-      - uses: codecov/codecov-action@v1
+          export PYTHONWARNINGS="ignore:Certificate for localhost has no \`subjectAltName\`"
```

Comment:
Fine to leave but why is this needed?

Reply:
I just included it as part of the cut and paste from qp-klp's qiita installation. In this case I thought it better to remove it.